### ISSUE TITLE
allow any element

### ISSUE THREAD

**Issue by BetterCallSky** [dantrain/react-stonecutter#42]
Currently, if you provide custom react components under `<CSSGrid>`, it will throw an error.
The code is here https://github.com/dantrain/react-stonecutter/blob/master/src/utils/assertIsElement.js

The solution is to wrap it with an extra `<div>`. Unfortunately, it doesn't work well for me because I also use `react-dnd`, and I have a weird bug on mobile.

Is possible to remove this validation? I am using a fork with this change and it works without any problems.
The custom component must only apply provided `style` to the main `<div>`.

**Comment by dantrain** [comment]
Yes I think this could do with a rethink. Perhaps an API closer to React Motion's render prop would be better? Anyway I will see about allowing switching the validation off and documenting the use of the provided `style` prop.
